Forms polyfill (#74)
CoherentLabs/GameUIComponents: components/form-control/README.md

* The data from `<input type="submit"/>` will be included in the request if this input is clicked.
* The **name** attribute is required if you want your input to be included in the form data on submitting!

## Select

> Native `<select>` element is not supported by Gameface. However, there is a polyfill that transforms `<select>` into a working dropdown inside the Gameface. That dropdown element - `<custom-select>` is supported by the form control component.

Reviewer: 1. in~side~ the Gameface Player
2. in~side~ Gameface

Author: Fixed

> The polyfill could be found inside the Gameface package - `Samples/uiresources/Dropdown`.

Reviewer: ```suggestion
The polyfill is located inside the Gameface package - `Samples/uiresources/Dropdown`.
```

Author: Fixed
